src/System.ServiceModel.Syndication/ref/System.ServiceModel.Syndication.cs

```diff
@@ -434,6 +427,9 @@ public abstract partial class SyndicationFeedFormatter
     {
         protected SyndicationFeedFormatter() { }
         protected SyndicationFeedFormatter(System.ServiceModel.Syndication.SyndicationFeed feedToWrite) { }
+        public System.Func<string, string, string, System.DateTimeOffset> DateTimeParser { get { throw null; } set { } }
+        public System.Func<string, string, string, string> StringParser { get { throw null; } set { } }
+        public System.Func<string, System.UriKind, string, string, System.Uri> UriParser { get { throw null; } set { } }
         public System.ServiceModel.Syndication.SyndicationFeed Feed { get { throw null; } }
```

Comment:
Do we really need the three new APIs? They are not available on desktop now. Would a partial façade be able to support them on desktop?

Reply:
New APIs need to be proposed and reviewed: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md

Reply:
@shmao, because we've had problems with parsing datetime, we looked at all the parsing code in the codebase and it turns out that the only things we attempt to parse are strings, uri's and datetime's. If we're going to expose overriding the datetime parsing, then it makes sense to expose the others.
@justinvp, when this project was started we didn't know where the final result would land. We had options which included it being a completely new package with a new namespace outside of CoreFx/Full framework, being a part of CoreFx, being a part of WCF and we even considered maybe we should talk to the ASP.NET core team about it living there. As we had an intern working on this, we didn't want to hold him back while we worked that out. The functionality of these new API's are absolutely needed, it's simply a case of what they look like that we need to work out. For example, we could convert the async methods to extension methods and make them work on the full framework too. We're still working out exactly what the best approach will be re compatibility.